capstone-frontend/src/components/Home.jsx

```diff
@@ -0,0 +1,304 @@
+                    })}
+                  </>
+                  :
+                  <div className='text-center'>
+                    <h3 className="mt-5">No Booklists Found</h3>
+                    <CreateList updateBookLists={this.props.updateBookLists} btnStyle='btn btn-primary mt-4' />
+                  </div>}
+              </>
+            }
+          </Tab>
+
+          <Tab eventKey='Booklists - Collaborator' title='Booklists - Collaborator' >
+
+            {(this.state.fetchingBookListsShared || this.props.fetchingCollabBookLists) ?
+              <div className='text-center mt-4'>
+                <Spinner animation='border' role='status' variant='primary' />
+              </div>
+              :
+              <>
+                {this.state.sharedBookListsMap.size > 0 ?
+                  <>
+                    {[...this.state.sharedBookListsMap.keys()].map(bookList => {
+
+                      const books = this.state.sharedBookListsMap.get(bookList);
+
+                      return (
+                        <Card key={bookList.id} className='m-4'>
+
+                          <Link to={`/listpage/${bookList.id}`}>
+                            <Card.Header>
+                              <Card.Title id='booklist-card-font-size-title' className='text-muted'>{bookList.name}</Card.Title>
+                            </Card.Header>
+                          </Link>
+
+                          {books.length !== 0
+                            ?
+                            <div id='booklist-content' className='m-auto'>
+                              {books.map(book =>
+                                <Card.Body key={book.id} >
+                                  <BookDescriptionOverlay book={book}>
+                                    <Card.Img className='img-fluid my-1 light-gray-border book-img-lg' variant='top' src={book.thumbnailLink} />
+                                  </BookDescriptionOverlay>
+                                  <Card.Title id='booklist-card-book-title-font-size' className='mt-4 mb-1 text-center book-img-text-width'>{book.title}</Card.Title>
+                                </Card.Body>)}
+                            </div>
+                            :
+                            <Card.Body>
+                              <Card.Text>Booklist contains no books</Card.Text>
+                            </Card.Body>}
+
+                          <Card.Footer>
+                            <Card.Text id='booklist-card-font-size-footer' className='text-muted'>{bookList.collaboratorsIDs.length} Collaborators</Card.Text>
+                          </Card.Footer>
+                        </Card>
+                      )
+                    })}
+                  </>
+                  :
+                  <h3 className="text-center mt-5">No Shared BookLists Found</h3>}
+              </>
+            }
+          </Tab>
+        </Tabs>
+      </div>
+    );
+  }
+}
+
+export default Home;
```

Comment:
nit: Make this a named export by putting export next to the class definition at the top

Reply:
Ok